src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/workbook/viewmodel/WorkbookViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package org.wycliffeassociates.otter.jvm.app.ui.workbook.viewmodel
+
+import javafx.beans.property.SimpleObjectProperty
+import javafx.beans.property.SimpleStringProperty
+import org.wycliffeassociates.otter.common.data.workbook.Chapter
+import org.wycliffeassociates.otter.common.data.workbook.Chunk
+import org.wycliffeassociates.otter.common.data.workbook.Workbook
+import tornadofx.*
+import java.io.File
+import java.lang.IllegalStateException
+
+class WorkbookViewModel: ViewModel() {
+    val activeWorkbookProperty = SimpleObjectProperty<Workbook?>()
+    val workbook: Workbook
+        get() = activeWorkbookProperty.value ?: throw IllegalStateException("Workbook is null")
```

Comment:
Rather than performing null (or ?.let) checks everywhere and dirtying up the code, I elected to make these getters throw errors if the value is null.
Since all of the getters in WorkbookViewModel follow this pattern, there should be no surprises as long as one knows this is the expected behavior.

Reply:
Update 6/26: Unfortunately I don't think the pattern of throwing an exception on null can apply to every property since Chunk is actually nullable (sometimes we are recording material about a Chapter, not a particular Chunk.)